### DISCUSSION TITLE
how to configure PersistentVolume

### DISCUSSION THREAD

**Discussion by johnyts** [prometheus-operator/prometheus-operator#4004] [Q&A]
Hello, How do I set up PersistentVolume for Prometheus? I've been trying for 2 days and I don't know how?
I added PV and PVC but how to configure it in Prometheus? please help

**Comment by paulfantom** [comment]
You need to configure it via `storage` option in Prometheus CR. Here is an example: https://github.com/thaum-xyz/ankhmorpork/blob/master/apps/monitoring/manifests/prometheus/prometheus.yaml#L65-L75
